omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb

```diff
@@ -124,13 +124,15 @@
           {log_handlers, [
 %%% Log handling is disabled, because it is redundant (nginx also logs requests)
 %%% If debug logging is needed, this can be uncommented to start logging somewhat verbose logs
+%%%          <% if node['private_chef']['webmachine']['bookshelf']['enable_request_logging'] %>
```

Comment:
Rather than keeping this commented, what do you think about uncommenting it, changing the default config value to false, an moving the explanation over to the attribute file.

Reply:
yesss!!!